Forms/Access SVN - Options.ACF

```diff
@@ -2740,6 +2793,50 @@ Private Sub cmdUTF8Encoding_Click()
   DoCmd.Hourglass False
 End Sub
 
+Private Sub cmdUTF8FullEncode_Click()
+  Dim i As Long
+  Dim j As Long
+  Dim strFoldername As String
+  Dim strFilename As String
+  Dim arrSCCItems() As SVNITEM
+  Dim strType As String
+  
+  DoCmd.Hourglass True
+  For i = 0 To 6
+    Select Case i
+      Case 0: strType = "Forms"
+      Case 1: strType = "Reports"
+      Case 2: strType = "Macros"
+      Case 3: strType = "Queries"
+      Case 4: strType = "Tables"
+      Case 5: strType = "Toolbars"
+      Case 6: strType = "Extras"
+    End Select
+    strFoldername = Me.txtOutputLocation.Value & "/" & strType
+      
+    SCC_GetAllVersionsSubFolder strFoldername, arrSCCItems
+    
+    If (Not arrSCCItems) <> -1 Then ' check that the arr is not nothing before looping through it
+      For j = LBound(arrSCCItems) To UBound(arrSCCItems)
+        If arrSCCItems(j).svName <> "" Then
+          strFilename = appendSlash(strFoldername) & arrSCCItems(j).svName
+          If Not ConvFile_to_UTF8(strFilename) Then
+            MsgBox "Conversion of '" & strFilename & "' to UTF-8 has failed."
+            DoCmd.Hourglass False
+            Exit Sub
+          End If
+        End If
+      Next
+    End If
+  Next
+  
+  SetConfigValue "UTF-8 Encode All", True, Me.txtOutputLocation.Value
+    
+  Me.cmdRun.SetFocus
+  Me.cmdUTF8FullEncode.Enabled = False

```

Comment:
The 'convert encoding to UTF-8' button should also be disabled, as doing this implies reencoding ucs2 files. The 2 buttons should also be disabled when you open the form if `ConfigValue "UTF-8 Encode All"` is True.

Reply:
done

Reply:
You still need to do the second half of this